.gitignore

```diff
@@ -7,3 +7,9 @@ node_modules/
 .DS_Store
 site/node_modules/
 site/themes/docsy
+alb-rule-priority-generator.js
+dns-cert-validator.js
+dns-delegation.js
+enable-long-arns.js
```

Comment:
We want to keep these guys out of .gitignore so that they're picked up if we make changes to our custom resources